clientconfig.go

```diff
@@ -68,14 +68,10 @@ func ClientConfigFromReader(resolvconf io.Reader) (*ClientConfig, error) {
 			}
 
 		case "search": // set search path to given servers
-			c.Search = make([]string, len(f)-1)
-			for i := 0; i < len(c.Search); i++ {
-				c.Search[i] = f[i+1]
-			}
+			c.Search = append([]string(nil), f[1:]...)
```

Comment:
`string(nil)`? can't use `[]string{}` ?

Reply:
Idiomatic go is to use `nil` here. It needs the type of the slice, but not the actual slice. It’s straight out of the wiki: https://github.com/golang/go/wiki/SliceTricks#copy.